From a22ff2fbc0e44ff44d387c69461095580d0971e3 Mon Sep 17 00:00:00 2001
From: Changming Huang <jerry.huang@nxp.com>
Date: Wed, 3 Nov 2021 19:02:08 +0800
Subject: [PATCH 16/20] ivshmem demo: change dev_id to 2 for qoriq platforms

According to the calculation express in file inmates/demos/ivshmem-demo.c:
	target = d->id < 2 ? (d->id + 1) : 0
When dev_id is 1, the inmate cell will send the IRQ to device 2, instead
of root cell 0.

Signed-off-by: Changming Huang <jerry.huang@nxp.com>
---
 configs/arm64/ls1028a-rdb-inmate-demo.c | 2 +-
 configs/arm64/ls1043a-rdb-inmate-demo.c | 2 +-
 configs/arm64/ls1046a-rdb-inmate-demo.c | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configs/arm64/ls1028a-rdb-inmate-demo.c b/configs/arm64/ls1028a-rdb-inmate-demo.c
index 7a1786e4..fa5d5d24 100644
--- a/configs/arm64/ls1028a-rdb-inmate-demo.c
+++ b/configs/arm64/ls1028a-rdb-inmate-demo.c
@@ -121,7 +121,7 @@ struct {
 			.bdf = 0 << 3,
 			.bar_mask = JAILHOUSE_IVSHMEM_BAR_MASK_INTX,
 			.shmem_regions_start = 0,
-			.shmem_dev_id = 1,
+			.shmem_dev_id = 2,
 			.shmem_peers = 1,
 			.shmem_protocol = JAILHOUSE_SHMEM_PROTO_UNDEFINED,
 		},
diff --git a/configs/arm64/ls1043a-rdb-inmate-demo.c b/configs/arm64/ls1043a-rdb-inmate-demo.c
index 53a65e3c..5f895022 100644
--- a/configs/arm64/ls1043a-rdb-inmate-demo.c
+++ b/configs/arm64/ls1043a-rdb-inmate-demo.c
@@ -131,7 +131,7 @@ struct {
 			.bdf = 0 << 3,
 			.bar_mask = JAILHOUSE_IVSHMEM_BAR_MASK_INTX,
 			.shmem_regions_start = 0,
-			.shmem_dev_id = 1,
+			.shmem_dev_id = 2,
 			.shmem_peers = 1,
 			.shmem_protocol = JAILHOUSE_SHMEM_PROTO_UNDEFINED,
 		},
diff --git a/configs/arm64/ls1046a-rdb-inmate-demo.c b/configs/arm64/ls1046a-rdb-inmate-demo.c
index a26d387a..35c8d955 100644
--- a/configs/arm64/ls1046a-rdb-inmate-demo.c
+++ b/configs/arm64/ls1046a-rdb-inmate-demo.c
@@ -132,7 +132,7 @@ struct {
 			.bdf = 0 << 3,
 			.bar_mask = JAILHOUSE_IVSHMEM_BAR_MASK_INTX,
 			.shmem_regions_start = 0,
-			.shmem_dev_id = 1,
+			.shmem_dev_id = 2,
 			.shmem_peers = 1,
 			.shmem_protocol = JAILHOUSE_SHMEM_PROTO_UNDEFINED,
 		},
-- 
2.25.1

